rs/sns/governance/src/governance.rs

```diff
@@ -5057,16 +5097,20 @@ impl Governance {
         }
     }
 
+    // This is a test-only function, so panicking should be okay.
     pub fn advance_target_version(
         &mut self,
         request: AdvanceTargetVersionRequest,
     ) -> AdvanceTargetVersionResponse {
-        self.push_to_upgrade_journal(upgrade_journal_entry::TargetVersionSet::new(
-            self.proto.target_version.clone(),
-            request.target_version.clone(),
-        ));
+        let AdvanceTargetVersionRequest {
```

Comment:
Can you add `self.check_test_features_enabled();` here? It will panic but I think that's okay because no one should be calling this yet